scripts/v3.12/php-8.1/argon2/APKBUILD

```diff
@@ -32,14 +32,14 @@ package() {
     make DESTDIR="$pkgdir" install || return 1
 
     # A version number is not appended to the shared library file by default, so we do it ourselves
-    mv "$pkgdir"/usr/lib/libargon2.so "$pkgdir"/usr/lib/libargon2.so.$_soname || return 1
-    ln -s libargon2.so.$_soname "$pkgdir"/usr/lib/libargon2.so || return 1
-	ln -s libargon2.so.$_soname "$pkgdir"/usr/lib/libargon2.so.${_soname%%.*} || return 1
+    mv "$pkgdir"/usr/lib/x86_64-linux-gnu/libargon2.so "$pkgdir"/usr/lib/x86_64-linux-gnu/libargon2.so.$_soname || return 1
```

Comment:
i'm sure there are some variables available but only found `$CARCH`. i dont know how to get the `linux-gnu` part from variables